casper/src/main/rholang/StopGap.rho

```diff
@@ -0,0 +1,26 @@
+new builtinsStore, activeStore in {
+  builtinsStore!({}) |
+  activeStore!(true) |
+  contract @(*StopGap, "set")(@title, @value, ack) = {
```

Comment:
First, where does the `StopGap` name come frome? I don't see it created/passed anywhere.

----------
I don't quite get how this will work. 
```
new StopGap(`rho:stopGap`) in { … } 
```
will return unforgeable name that  will reference `StopGap` contract. We can then use contracts that are leaving under this name to register new common contracts (like `("ListOps", "fold")` ) and use them if needed. The `activeStore` flag indicates wheter the contract map living under `StopGap` name can be mutated (i.e. if new contracts can be added) and anyone who has access to the name. It  means that anyone can "close" the map and anyone can  replace the contract leaving under the name. What am I missing here?

Reply:
Ugh, I missed the notification from this due to a bad filter.  I wrote you on discord, but it looks like you missed that.

Yeah, StopGap.rho isn't quite right.  In fact, I just realized that the `(*name, "method")` idiom can't be made secure if you expose `name`.  I'll have to redo `StopGap.rho` and change `MakeMint.int2NN` to use a different pattern.

The intent was that code in the genesis block could install these contracts in a way that they could be looked up by name, but nothing running after the genesis block could install contracts this way.  The idea was that code in StopGap.rho would be moved into scala to provide the "injection" syntax.  I also wanted to make it so that each piece of code could run independently, since that's how it currently works.

The alternative you suggested is to install each process under its own public key.  However, the particular name that is generated in the `new` will depend on the timestamp of the deployment, so I can't just write rholang code.  I would need to figure out how the code is deployed in the genesis block, then write scala code to get the timestamp for the deployment, store all the private keys somewhere to generate the signatures, and generate the names that would be used, then inject all that stuff into the genesis block somehow.  This is much more complicated than being able to use the package API we discussed.  Is there a ticket for writing up that part of the registry?